server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -83,6 +83,14 @@ export class RestWrapper {
                     debug(`request to ${options.url} failed ${error ? error.message : ""}`);
                 }
 
+                if (error.response && error.response.status
+                    && error.response.data && error.response.data.retryAfter
+                    && error.response.status === 429 && error.response.data.retryAfter > 0) {
```

Comment:
This could be simplified using [optional chaining](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining) and the knowledge that `(undefined === 429) === false` and `(undefined > 0) === false`